tests/unit/services/getAppInfo.js

```diff
@@ -7,12 +7,15 @@ const pkgJSON = require('../../../package.json');
 chai.should();
 
 describe('getAppInfo', () => {
-  it('responds with correct app information', () => getAppInfo()
+  const commit = 'abc123';
+  const config = { herokuSlugCommit: commit };
```

Comment:
Another option would be to use https://www.npmjs.com/package/proxyquire in the tests. That way we wouldn't need to pass `config` around. Thoughts?

Reply:
In general you want to inject config instead of grabbing it from some external state that must be shimmed in testing. But in this one case I would actually suggest reading `HEROKU_SLUG_COMMIT` directly from `process.env` instead of including it with the rest of the config. Git hashes and Heroku hashes aren't really part of your application proper, they're more like metadata.